supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributes.scala

```diff
@@ -49,13 +50,16 @@ object PipelinesApiConfigurationAttributes {
 
   final case class VirtualPrivateCloudConfiguration(name: String, subnetwork: Option[String], auth: GoogleAuthMode)
   final case class BatchRequestTimeoutConfiguration(readTimeoutMillis: Option[Int Refined Positive], connectTimeoutMillis: Option[Int Refined Positive])
+  final case class MemoryRetryConfiguration(errorKeys: List[String], multiplier: Double)
```

Comment:
Should this be propagated as `multiplier: Double Refined Positive`, such that the multiplier type is always positive throughout the code?

Reply:
Oh I somehow missed this comment. I will update the code accordingly and hit the recycle button then. Thanks!

Reply:
So the `multiplier` at this point, which is when Cromwell is started and Papi backend is initialized, is validated that it is a positive double, but not used. It is used in the [StandardAsyncExecutionActor](https://github.com/broadinstitute/cromwell/blob/1717c0311218d5a1117bb94ce4dd89163b39c948/backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala#L188) when a task has failed and the `retryWithMoreMemory` condition is true. I am not revalidating again because if it was not a valid double, Cromwell would have thrown an error when it was started and validated in the Papi backend. Does that make sense? I can remove the `multiplier` field from `MemoryRetryConfiguration` altogether to make it easy.

Reply:
I can also revalidate that the multiplier is a positive double again at [StandardAsyncExecutionActor](https://github.com/broadinstitute/cromwell/blob/1717c0311218d5a1117bb94ce4dd89163b39c948/backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala#L188), but if at this point it is not a valid double, we can switch the multiplier value to default (which is 2) or capture the error and fail the task without retrying.

Reply:
After talking and coding f2f, the code has been updated to use `Double Refined Positive` type for multiplier all the way through.